packages/sitecore-jss/src/feaas/themes.ts

```diff
@@ -0,0 +1,89 @@
+import {
+  ComponentRendering,
+  HtmlElementRendering,
+  LayoutServiceData,
+  RouteData,
+  getFieldValue,
+} from '../layout';
+
+/**
+ * Pattern for library ids
+ * @example -library--foo
+ */
+const FEAAS_LIBRARY_ID_REGEX = /-library--([^\s]+)/;
+
+/**
+ * FEAAS CDN hostnames
+ */
+export const FEAAS_CDN = Object.freeze({
+  PRODUCTION: 'https://feaas.blob.core.windows.net',
+  STAGING: 'https://feaasstaging.blob.core.windows.net',
```

Comment:
Ah, I see now why you have exported `FEAAS_CDN` all the way up through the framework package (because we may need to configure for staging). But I'm wondering, is this something we need to support? Or can we always just use production for our SDK? @Inviz maybe more a question for you.

Reply:
It's a good question, a valid one. Currently we can get away with just using the production, but ideally we should be able to use Staging for the rendering host in pages. We actually need to do the same for components too and we are not doing it just yet, but only because we probably have to do it in JSS. So it's perhaps a good time to do that. 

This staging/production distinction is for static files like html and css. The SDK itself is a _third_  hostname (feaasstatic), for which there's no environmental distinction.

Reply:
Thanks for clarification